install/kubernetes/helm/istio/charts/sidecar-injector/templates/deployment.yaml

```diff
@@ -52,4 +53,5 @@ spec:
           name: istio-inject
           items:
           - key: config
-            path: config
\ No newline at end of file
+            path: config
+{{- end }}
```

Comment:
super nit: can you add the missing newline (and check it still works after that - sometimes concat is off)